engine/src/flutter/shell/common/shell.h

```diff
@@ -45,6 +45,13 @@
 
 namespace flutter {
 
+struct SizeConstraints {
+  double min_width = 0.0;
+  double max_width = 0.0;
+  double min_height = 0.0;
+  double max_height = 0.0;
+};
```

Comment:
FYI, `//shell/platform/common` has: https://github.com/flutter/flutter/blob/f6bbf765240c19fbd70afd47206cece25b9be5fa/engine/src/flutter/shell/platform/common/geometry.h#L86-L103

I'd consider moving that down to `//fml/geometry.h` or something